recipes/mosquitto/2.x/conanfile.py

```diff
@@ -0,0 +1,136 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class Mosquitto(ConanFile):
+    name = "mosquitto"
+    license = "EPL-2.0"
```

Comment:
https://github.com/eclipse/mosquitto/blob/ec1cad087067a5ce0d825e33de0da77798b671da/LICENSE.txt#L2

```suggestion
    license = "EPL-2.0", "EPL-1.0"
```